.github/workflows/ci.yml

```
@@ -1012,264 +1012,54 @@ jobs:
-    needs: [
-      job_setup,
-      job_lint,
-      job_unit-tests
-    ]
-    name: Publish @tryghost/admin-x-activitypub
-    runs-on: ubuntu-latest
+    uses: ./.github/workflows/publish-package.yml
+    with:
+      package-name: '@tryghost/admin-x-activitypub'
+      package-path: apps/admin-x-activitypub
+      node-version: ${{ env.NODE_VERSION }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)